docs/site/src/app/pages/partners/partners.component.html

```diff
@@ -7,6 +7,36 @@
   selected partner agencies in the world. All of them are licensed agencies and having official Spica tests."
   subtitle="Don’t hesitate to contact them for:"
 >
+  <div fxLayout="row" fxFlex="100%" fxLayoutAlign="flex-end">
+    <div fxFlex.gt-sm="25%" fxflex.lt-md="100%" fxLayoutAlign="center center">
+      <div fxFlex="75%">
+        <mat-form-field appearance="outline">
+          <mat-label>Select from country for see partner.</mat-label>
+          <mat-select name="country" [(value)]="selectedCountry">
+            <mat-option *ngFor="let country of countryList " [value]="country.key">
+              {{ country.value }}
+            </mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+      <div fxFlex="25%" class="partner-search">
+        <button mat-flat-button color="primary" class="btn" (click)="countrySelect()">
+          <mat-icon>search</mat-icon>
+        </button>
+      </div>
+    </div>
+  </div>
+
+  <div fxFlex="100">
+    <div fxFlex="60" fxLayout="row wrap" fxLayoutAlign="start" fxLayoutGap="5px grid">
+      <div fxFlex.lt-md="100%" fxFlex="50%" *ngFor="let item of partnerFilter">
+        <a href="{{ item.url }}" target="_blank">
```

Comment:
Please use `[href]="item.url"` or `[attr.href]="item.url"` to follow our templating conventions.